extras/parse_travis_log.py

```diff
@@ -638,55 +716,51 @@ def printable_summary(list_of_changed_files,
-    :param status_cmake_configure:  Status of the 'CMake configure':
-                                    True, False or None
-    :param status_make:             Status of the 'make':
-                                    True, False or None
-    :param status_make_install:     Status of the 'make install':
-                                    True, False or None
-    :param status_amazon_s3_upload: Status of the Amazon S3 upload:
-                                    True, False
-    :param status_tests:            Status of the test suite run:
-                                    True, False or None
-    :param summary_vera:            Dictionary of dictionaries of VERA++
-                                    messages per file.
-    :param summary_cppcheck:        Dictionary of dictionaries of cppcheck
-                                    messages per file.
-    :param summary_format:          Dictionary of dictionaries of clang-format
-                                    messages per file.
-    :param summary_pep8:            Dictionary of dictionaries of PEP8
-                                    messages per file.
-    :param summary_errors:          Dictionary of build error messages.
-    :param summary_warnings:        Dictionary of build warning messages.
-    :param number_of_errors:        Number of errors.
-    :param number_of_warnings:      Number of warnings.
-    :param number_of_tests_total:   Number of tests total.
-    :param number_of_tests_failed:  Number of tests failed.
-    :param exit_code:               Build exit code: 0 or 1
-    :return:                        Formatted build summary string.
-    """
-    header = '\n+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ' \
-             '+ + + + + + + + + +' + \
-             '\n+                                                           ' \
-             '                  +' + \
-             '\n+           N E S T   T r a v i s   C I   B u i l d   S u m ' \
-             'm a r y           +' + \
-             '\n+                                                           ' \
-             '                  +' + \
-             '\n+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ' \
-             '+ + + + + + + + + +'
-
-    build_summary = header + '\n\n'
+
+    header_l1 = '\n+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +'  # noqa
+    header_l2 = '\n+                                                                             +'  # noqa
+    header_l3 = '\n+           N E S T   T r a v i s   C I   B u i l d   S u m m a r y           +'  # noqa
+    header_l4 = '\n+                                                                             +'  # noqa
+    header_l5 = '\n+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +'  # noqa
+
+    build_summary = header_l1 + header_l2 + header_l3 + header_l4 + header_l5 + '\n\n'               # noqa
 
```

Comment:
@jougs: Thanks for the Python multi-line hint!  I finally made the layout of the header smaller. 4292033 The underlaying problem is the pep8 E501 rule limiting the line size to 79. Neither continuation lines nor multi-line comments allow '# noqa' to suppress pep8 checking. 
In my opinion this is a strong limitation which makes the code less readable. Readability is important for maintainability. As a result this limitation impacts code quality.